verde/utils.py

```diff
@@ -281,3 +282,64 @@ def grid_to_table(grid):
     coord_dict.update(var_dict)
     data = pd.DataFrame(coord_dict)
     return data
+
+
+def load_surfer(fname, dtype='float64'):
+    """
+    Read data from a Surfer ASCII grid file.
+
+    Surfer is a contouring, griding and surface mapping software
+    from GoldenSoftware. The names and logos for Surfer and Golden
+    Software are registered trademarks of Golden Software, Inc.
+
+    http://www.goldensoftware.com/products/surfer
+
+    This feature was ported from Fatiando, but instead of
+    output being a dictionary, in verde the output is a `xarray.DataArray`
+    that contains the easting and northing coordinates, and data values.
```

Comment:
The mention of Fatiando isn't necessary here. But it is good to describe what is returned by the function and some other details (like the masking of NaNs and inclusion of metadata in `attrs`).